### PR TITLE
chore(router): Cleanup standalone router charts

### PR DESCRIPTION
**Pull request by krancour**
Includes:
- `router` chart (alpha) goes away; gets replaced with `router-beta1-test`
- `router-beta1-test` manifests are copies of relevant manifests from `workflow-beta1-test`
- Manifest for `deis` namespace gets added to both `router-beta1-test` and `router-dev` charts.  These "standalone" router charts are mutually exclusive with installing all of Workflow-- so these charts must account for namespace creation themselves.  It was an oversight that they did not previous and it went undetected because some clusters automatically create namespaces, depending on what admission controls are in place.

### CODE REVIEW

File: router-beta1-test/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-name: router
+name: router-beta1-test
 home: https://github.com/deis/router
-version: 2.0.0-alpha
+version: v2-beta
```
Comment (arschles) [inline router-beta1-test/Chart.yaml:3]: are we calling this beta `v2-beta` across the board?

File: router-beta1-test/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Deis Router 2.0.0-alpha
+# Deis Router 2.0.0-beta
 
 WARNING: this chart is for testing only! Features may not work and there are likely to be bugs.
```
Comment (arschles) [inline router-beta1-test/README.md:3]: maybe change to indicate beta status, instead of testing only?

File: router-beta1-test/Chart.yaml (hunk 1 above)
Comment (krancour) [inline router-beta1-test/Chart.yaml:3]: Good question.  I just emulated what the `workflow-beta1-test` chart had in this same field, with the assumption that it was correct.

File: router-beta1-test/README.md (hunk 2 above)
Comment (krancour) [inline router-beta1-test/README.md:3]: Again... I just followed the pattern from the `workflow-beta1-test` chart with the assumption that it was correct.

File: router-beta1-test/Chart.yaml (hunk 1 above)
Comment (arschles) [inline router-beta1-test/Chart.yaml:3]: K, sounds like there's consistency, so I'm good with this

File: router-beta1-test/README.md (hunk 2 above)
Comment (arschles) [inline router-beta1-test/README.md:3]: Same here regarding consistency.

Comment (jchauncey) [inline router-beta1-test/README.md:3]: So I made logger v2.0.0-beta1 just FYI

**Comment by krancour** [comment]
@arschles thanks for the review.  fwiw... I think your questions are broadly applicable to the multiple `*-beta1-test` charts we have atm... perhaps it's worth opening a serparate issue to get some clarification on it and/or track the fix if needed.

**Comment by arschles** [comment]
@krancour good idea - created https://github.com/deis/charts/issues/184 for naming consistency

**Pull request merged**